pywbem_mock/_dmtf_cim_schema.py

```diff
@@ -0,0 +1,426 @@
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+#
+"""
+A DMTF CIM schema is the collection of CIM qualifier declarations and CIM
+classes managed by the DMTF and released as a single tested, coherent package
+with a defined major, minor, update version number.  The MOF representation of
+a DMTF schema release is packaged as a single zip file and is available on the
+DMTF web site.
+
+Because the DMTF MOF schemas are the source of most of the classes that a WBEM
+server would implement we felt it was important to create a simple mechanism
+to include the DMTF CIM classes and qualifier declarations in the
+:class:`~pywbem.FakedWBEMConnection` mock repository.
+
+The :class:`~pywbem_mock.DMTFCIMSchema` class downloads the DMTF CIM schema MOF
+(a zip file) defined by version from the DMTF web site into a defined directory,
+unzips the MOF files into a subdirectory and makes the this subdirectory
+available as a property of the :class:`~pywbem_mock.DMTFCIMSchema` object.
+
+Multiple DMTF CIM schemas may be maintained in the same `schema_root_dir`
```

Comment:
On the sentence "Multiple DMTF CIM schemas...": At this point, the parameter `schema_root_dir` is not known yet. It seems to me it is dificult to find the right language to describe this detail, and also it only matters once the reader is at the level of single parameters. I suggest to remove this sentence from this overview section.

Reply:
DONE. But we need to state it and I moved it to the parameter schema-root_dir